src/main/java/me/mrCookieSlime/Slimefun/hooks/github/ContributionsConnector.java

```diff
@@ -53,13 +61,16 @@ public String getURLSuffix() {
 	private void computeContributors(JsonArray array) {
 	    for (int i = 0; i < array.size(); i++) {
 	    	JsonObject object = array.get(i).getAsJsonObject();
-	    	
+
 	    	String name = object.get("login").getAsString();
 	    	int commits = object.get("contributions").getAsInt();
 	    	String profile = object.get("html_url").getAsString();
-	    	
-	    	if (!blacklist.contains(name)) {
-	    		Contributor contributor = SlimefunPlugin.getUtilities().contributors.computeIfAbsent(name, key -> new Contributor(name, profile));
+
+	    	if (!name.toLowerCase().contains("bot") && !blacklist.contains(name)) {
+	    		Contributor contributor = SlimefunPlugin.getUtilities().contributors.computeIfAbsent(
+	    				name,
+						key -> new Contributor(aliases.getOrDefault(name, name), profile)
```

Comment:
The alias should better be treated seperately.

We want the GitHub name to display correctly...

Reply:
Hmm, not sure I fully agree. I mean, it's showing the contributors but displaying their heads. Having their MC names there so they can be identified makes more sense. Especially since you can click on it to get their GitHub profile anyway.

Imagine someone just having that skin and that name in MC, they didn't contribute but it looks like they did now. Doubt anyone would do that just to show off but... people are weird.

Reply:
Then add a "(MC: username)" or whatever to it if they differ.

But replacing the name is just an incorrect Model.